packages/circle-react-elements/src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,50 @@
+import { ArrowUpRight, CheckCircle } from 'lucide-react';
+
+import { Button } from '~/components/ui/button';
+
+export interface NotificationProps {
+  /** The wallet */
+  onClose?: () => void;
+  description: string | React.ReactElement;
+  externalLink?: string;
```

Comment:
Regarding props, instead of limiting the description to a ReactElement, we could generalize it by accepting `children: React.ReactNode`. This way, we allow any content to be placed under the title and success icon. In this case, we wouldn’t need the `externalLink` prop either.
So to sum up - I suggest to replace `description` and `externalLink` with `children`